mail/dovecot2/Portfile

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github                      1.0
-PortGroup           cxx11                       1.1
+PortGroup           github  1.0
+PortGroup           cxx11   1.1
 
-github.setup        dovecot core 2.3.7.1
+github.setup        dovecot core 2.3.7.2
+revision            0
```

Comment:
Needs to be rebased against master since there was a rev bump.

Reply:
so what should revision then be set to ?

Reply:
0, because a new version is also being set. The diff just has to be changing revision 1 to revision 0, rather than changing from no revision line to revision 0.

Reply:
there's no revison line in the original Porfile, so my diff is correct